product/roundhouse.core/migrators/DefaultDatabaseMigrator.cs

```diff
@@ -255,12 +255,23 @@ public IEnumerable<string> get_statements_to_run(string sql_to_run)
 
         public void record_script_in_scripts_run_table(string script_name, string sql_to_run, bool run_this_script_once, long version_id)
         {
+            var hash = hash_generator.create_hash(sql_to_run, true);
+            if (!configuration.DoNotStoreScriptsRunText)
```

Comment:
Isn't this one inverted? Shouldn't the `sql_to_run` be set to `null` if `DoNotStoreScriptsRunText` is set to `true`?